### PR TITLE
INT-13 Dark/Light Mode implemented, fixed offset applicant cards on s…

### PR DESCRIPTION
**Pull request by PH86**
…ingle job page

### CODE REVIEW

File: src/components/DashCard/DashCard.css [hunk 1]
```diff
@@ -1,12 +1,10 @@
-
 .dashcard-container{
     margin: 0 auto;
     padding: 0.5rem;
     margin: 0.5rem;
     height: fit-content;
-    color: var(--clr-text-secondary);
     border-radius: var(--radius);
-    box-shadow: var(--dark-shadow-inverse);
+
```
Comment (JackHewitt08) [inline src/components/DashCard/DashCard.css]: Remove extra line

File: src/components/Graph/Graph.tsx [hunk 2]
```diff
@@ -46,7 +52,7 @@ export const DashboardLineGraph: React.FC<ILineGraph> = ({
 	dataKeyArea,
 }): React.ReactElement => {
 	return (
-		<div className="graph-container">
+		<Wrapper className="graph-container">
```
Comment (JackHewitt08) [inline src/components/Graph/Graph.tsx]: Better name for this

**Review by JackHewitt08 — APPROVED** [review]

File: src/components/Applicant/ApplicantCard.tsx [hunk 3]
```diff
@@ -14,15 +16,19 @@ export interface IApplicantCard {
     cvLink: string,
 }
 
+const Article = styled.article`
```
Comment (sambuckley1995) [inline src/components/Applicant/ApplicantCard.tsx]: Not sure how I feel about naming the styled component the same as its semantic counterpart

File: src/components/DashCard/DashCard.tsx [hunk 4]
```diff
@@ -1,16 +1,22 @@
...
+import styled from "styled-components";
+import { shadow } from '../../themes/theme';
 
 interface IDashCard {
     title: string,
     content: number | string,
 }
 
-export const DashCard: React.FC<IDashCard> = ({title, content}): React.ReactElement => {
+const Article = styled.article`
```
Comment (sambuckley1995) [inline src/components/DashCard/DashCard.tsx]: Same as above with naming

File: src/components/Applicant/ApplicantCard.tsx (hunk 3 above)
Comment (PH86) [reply src/components/Applicant/ApplicantCard.tsx]: I will change it to Card, that is probably a better description of it

File: src/components/Sidebar/Sidebar.tsx [hunk 5]
```diff
@@ -21,18 +29,26 @@ export const Sidebar: React.FC<{}> = (): React.ReactElement => {
...
     });
-  }, []);
+  }, [theme]);
+
+  const Sidebar = styled.div`
+    background-color: ${sidebarBackgroundColor};
+    color: ${sidebarTextColor};
+  `;
+
+  const Ul = styled.ul`
```
Comment (sambuckley1995) [inline src/components/Sidebar/Sidebar.tsx]: same as above with naming

**Review by sambuckley1995 — APPROVED** [review]

**Review by sambuckley1995 — APPROVED** [review]

**Review by sambuckley1995 — APPROVED** [review]

**Review by sambuckley1995 — APPROVED** [review]

**Pull request merged**